portfolio/src/main/webapp/script.js

```diff
@@ -56,6 +56,13 @@ function loadSkills () {
  */
  async function fetchFromData() {
   const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('secret-message').innerText = quote; 
+  // This gives a list of messages
+  const msgs = await response.json();
+  console.log(msgs); 
```

Comment:
nit: I know it says in the instruction to console.log the msgs to see, but in general when submitting a PR it's good practice to remove any log messages that are purely for debugging.

Reply:
oh yea I'm so sorry for missing that totally forgot to take it out